sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -68,6 +68,16 @@ class ParquetFileFormat
 
   override def toString: String = "Parquet"
 
+  private def verifySchema(schema: StructType): Unit = {
+    if (schema.size < 1) {
```

Comment:
any particular reason for using "< 1" and not "== 0". just curious

Reply:
no particular reason :-). I can change that to "== 0".